external/googletest

```diff
@@ -0,0 +1 @@
+../../../third_party/googletest/
```

Comment:
This 'external/googletest' file was incorrectly added to the comit.
On my machine it's a symlink with those contents.